solutions/standard/version.tf

```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = "1.66.0"
     }
+    time = {
+      source  = "hashicorp/time"
+      version = ">= 0.9.1, < 1.0.0"
```

Comment:
DAs should lock into an exact provider version

Reply:
resolved